pywbemtools/pywbemcli/pywbemcli.py

```diff
@@ -265,7 +265,7 @@ def validate_connections_file(connections_repo):
                    'definitions. The default if this option is not specified '
                    'is the file name "{df}" in the users home directory. '
```

Comment:
The variable shown is no longer just the file name, but the full file path. What we still want to get across is that that directory is the home directory, because it will change for each user.

I suggest changing the text to something like:
```
    'is the file "{df}" (in the users home directory). '
```

Reply:
DONE